Core/SyncBookmarksAdapter.swift

```diff
@@ -120,6 +130,13 @@ public final class SyncBookmarksAdapter {
         self.provider = provider
     }
 
+    static private func showErrorAlert() {
```

Comment:
Better name for this would be `notifyBookmarksSyncLimitReached`, because this function doesn't show anything by itself, only sends a notification. Same applies to Credentials.